fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveConnectorInternalMgr.java

```diff
@@ -111,7 +110,7 @@ public void shutdown() {
 
     public IHiveMetastore createHiveMetastore() {
         // TODO(stephen): Abstract the creator class to construct hive meta client
-        HiveMetaClient metaClient = HiveMetaClient.createHiveMetaClient(properties);
+        HiveMetaClient metaClient = HiveMetaClient.createHiveMetaClient(hdfsEnvironment, properties);
```

Comment:
If we using hdfsEnvironment here, why still need properties?

Reply:
Because we have to pass configuration.